src/components/Editor/Preview/Popup.js

```diff
@@ -72,13 +87,18 @@ export class Popup extends Component<Props> {
     }
   }
 
-  getChildren(root: Object, getObjectProperties: Function) {
-    const actors = {};
+  getChildren(root: Object) {
```

Comment:
is root of type Node? https://github.com/devtools-html/debugger.html/blob/master/src/utils/sources-tree/types.js#L14

Reply:
it is, thanks for pointing to the type